### ISSUE TITLE
Broken with ring 1.9.1

### ISSUE THREAD

**Issue by thiru** [bhauman/figwheel-main#284]
I tried updating my ring dependencies from 1.8.2 to 1.91.  When doing so I get the following error

```
Execution error (ArityException) at figwheel.server.jetty-websocket/async-websocket-configurator$fn$fn (jetty_websocket.clj:116).
Wrong number of args (2) passed to: ring.adapter.jetty/async-proxy-handler
```

**Comment by bhauman** [comment]
I'll take a look.

**Comment by bhauman** [comment]
Deploying a fix to 0.2.13-SNAPSHOT

**Comment by thiru** [comment]
It works! Thank you!
